apps/frontend/app/(client)/(code-editor)/_components/EditorHeader.tsx

```diff
@@ -369,6 +449,14 @@ export default function Editor({
           </SelectContent>
         </Select>
       </div>
+      <BackCautionDialog
+        confrim={isModalConfrimed}
+        isOpen={showModal}
+        title="Leave this page?"
+        description="Changes you made my not be saved."
+        onClose={() => setShowModal(false)}
+        onBack={() => setStay(false)}
```

Comment:
```suggestion
        onBack={() => router.push(whereToPush.current as Route)}
```
stay 상태를 없앨 수 있을 것 같아요!